mscore/exportxml.cpp

```diff
@@ -3849,8 +3886,10 @@ static void directionMarker(Xml& xml, const Marker* const m)
       if (sound != "") {
             xml.stag("direction placement=\"above\"");
             xml.stag("direction-type");
-            if (type != "") xml.tagE(type);
-            if (words != "") xml.tag("words", words);
+            QString positioning = " ";
```

Comment:
I believe this is the culprit for the mtest failures, Seems the space gets added even if nothing follows